process/process.md

```diff
@@ -0,0 +1,291 @@
+documentation, comments and commit message.
+
+## Create Pull Request
+
+When your commits are ready, you can create a pull requests to
+appropriate repositories. We are hosting all our code on GitHub, so
+you should be familiar with the process. But, in short:
+
+1. Fork the repository into your account
+2. Clone it locally
+3. Make changes and create commits as described above
+4. Push changes back into your GitHub repository, preferably in a
+   separate branch
+5. Use this branch to create Pull Request into corresponding
+   Xen-Troops repository
+
+### Describe your Pull Request
+
+If your PR consists of one commit, GitHub will do this for you. But if
+there are more commits, you need to provide comprehensive description
+for the PR as a whole. The same rules apply: short and descriptive PR
+name, detailed description of a problem solved. Do not try to squash
+your commit messages, write description for the PR as one entity instead.
+
+If appropriate, write how you tested it. Wait for automated checks to run.
+
+### Check-list
+
+* Code conforms coding style
+* Documentation is up-to date
+* Commits are atomic
+* Commits have good description
+* Code checker / style checker is happy
+* PR description is in good shape
+
+## Review
+
+When you satisfied with your PR, as someone to review it. You can find
+list of maintainers and reviewers at
+[reviewers](https://xen-troops.github.io/reviewers) page. In most
+cases GitHub will assign reviewers immediately, but it is a good idea
+to ping reviewers to tell them that PR is really ready for review.
+
+Not every comment from reviewer require changes in code. But many of
+them are. If reviewer asks you a question, do not haste to change your
+code. Answer the question first.
+
+Every project has maintainer(s). Maintainers have full vision of
+project and its architecture. In most cases maintainers will support
+code you contributed for rest of its life. So maintainers can ask you
+to make changes into your PR so it better fit into project's
+architecture. Even if your code is 100% correct and bug-free, it does
+not mean that it will be accepted as is.
+
+When all comments are addressed, reviewers need to explicitly approve
+pull request by providing Reviewed-by or Acked-by tag.
+
+After collecting all needed R-b tags, you should include them
+into appropriate commit messages, rebase changes onto current upstream
+branch and force push changes into your local branch.
+
+## Merge
+
+PR is ready to be merged when:
+
+1. It is on top of the current upstream branch (or at least does not
+   have any conflicts with it)
+2. Has all needed tags (Signed-off-by, Reviewed-by:, etc)
+3. All pipelines are finished with success
+
+If you meet all of those requirements, you can ask integrator or
+maintainer to merge your PR.
+
+Please note, that we do not use merge commits. We follow rebase
+strategy.
+
+# Licensing
+
+All new code should be licensed under Apache-2.0 license by default.
+
+# "Temporary" changes
```

Comment:
Better to move this section above "Check list" and have the "Issue created for fixing temporary change" item added to the check list